control-plane/pkg/core/config/utils.go

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import (
+	"fmt"
+
+	eventing "knative.dev/eventing-kafka-broker/control-plane/pkg/apis/eventing/v1alpha1"
+)
+
+// ContentModeFromString returns the ContentMode from the given string
+func ContentModeFromString(mode string) ContentMode {
+	switch mode {
+	case eventing.ModeBinary:
+		return ContentMode_BINARY
```

Comment:
Why that weird naming? Could it be just `ContentModeBinary` (even if it's a const) ?

Reply:
They are generated by the protobuf compiler.